composables/useVerifyAccount.ts

```diff
@@ -12,7 +12,10 @@ const signMessagePolkadot = async (address: string, message: string) => {
   return signedMessage.signature
 }
 
-export const SIGNATURE_MESSAGE = 'Verify ownership of this account on Koda'
+const SIGNATURE_MESSAGE = 'Verify ownership of this account on Koda'
+
+export const generateSignatureMessageWithProfileVersion = (version: number) =>
```

Comment:
we might add the version system for other things , let's keep it generic here 

something like `generateVersionedSignatureMessage` or anything similar 

```suggestion
export const generateVersionedSignatureMessage = (version: number): string  =>
```

Reply:
I have changed the name. However, I do not prefer to add the version system for other things, as the version is stored in the profile. It should be only used with profile-related.